packages/plugin-hardhat/src/utils/options.ts

```diff
@@ -2,11 +2,15 @@ import { ValidationOptions, withValidationDefaults } from '@openzeppelin/upgrade
 
 export interface Options extends ValidationOptions {
   constructorArgs?: unknown[];
+  timeout?: number;
+  pollingInterval?: number;
```

Comment:
Something I learnt recently is that if we include docstrings for a field, they are shown when it's autocompleted in an IDE. Example:

https://github.com/OpenZeppelin/solidity-docgen/blob/ebae54721adfdc13cbef3141dda63a359133e21a/src/config.ts#L27-L31

![image](https://user-images.githubusercontent.com/481465/151575644-24eef4cc-b92d-41de-be13-3ce6beb50946.png)


So I think it'd be a good idea to start adding docstrings at least for the new options.